.env

```diff
@@ -0,0 +1,7 @@
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
```

Comment:
Are we going to change this at any point?

Reply:
We have some environment variables specified in travis so we can use a different password etc.